_data/simple-icons.json

```
@@ -11764,6 +11764,12 @@
             "source": "https://github.com/wasmCloud/branding/",
             "guidelines": "https://github.com/wasmCloud/branding/"
         },
+        {
+            "title": "wasmer",
```

I think it's spelled with a capital W.

```suggestion
            "title": "Wasmer",
```